vllm/v1/kv_cache_interface.py

```diff
@@ -206,9 +208,14 @@ def page_size_bytes(self) -> int:
         return page_size
 
     def max_memory_usage_bytes(self, vllm_config: VllmConfig) -> int:
-        # We allocate 1 block for each request now, so max_memory_usage_bytes is
-        # the same as page_size_bytes.
-        # Need to update this when supporting prefix caching.
+        if self.cache_strategy == "last":
+            # Keeps the last full block and one non-full block state:
+            return 2 * self.page_size_bytes
```

Comment:
will it be 3?
Say in chunked prefill, mamba_block_size 2, A means first step, B means second step. In the second step, we need  state at position a to compute state at position b and c.
```
AA AA BB BB B
    a     b c
```